src/modules/75-cf/pages/feature-flags/components/FeatureFlagsListing.tsx

```diff
@@ -0,0 +1,210 @@
+
+    const defaultVariation = getDefaultVariation(data)
+
+    const isFlagTypeBoolean = data.kind === FlagTypeVariations.booleanFlag
+    const typeToString = useFeatureFlagTypeToStringMapping()
+
+    return (
+      <Layout.Vertical>
+        <Layout.Horizontal>
+          <Text>
+            <VariationTypeIcon multivariate={data.kind !== FlagTypeVariations.booleanFlag} />
+            {getString(isFlagTypeBoolean ? 'cf.boolean' : 'cf.multivariate')}
+            {!isFlagTypeBoolean && (
+              <Text inline color={Color.GREY_400} padding={{ left: 'xsmall' }}>
+                ({typeToString[data.kind] || ''})
+              </Text>
+            )}
+          </Text>
+        </Layout.Horizontal>
+        {!hasCustomRules && (
+          <Container style={{ display: 'flex', alignItems: 'center' }}>
+            <VariationWithIcon
+              variation={defaultVariation}
+              index={data.variations.indexOf(defaultVariation)}
+              textStyle={{
+                fontSize: '12px',
+                lineHeight: '24px',
+                color: '#9293AB',
+                paddingLeft: 'var(--spacing-xsmall)'
+              }}
+              textElement={getString(isOn ? 'cf.featureFlags.defaultServedOn' : 'cf.featureFlags.defaultServedOff', {
+                defaultVariation: defaultVariation.name || defaultVariation.value
+              })}
+            />
+          </Container>
+        )}
+      </Layout.Vertical>
+    )
+  }
+
+  const columns: Column<Feature>[] = useMemo(
+    () => [
+      {
+        Header: getString('featureFlagsText').toUpperCase(),
+        accessor: row => row.name,
+        width: '40%',
+        Cell: (cell: Cell<Feature>) => (
+          <RenderFeatureFlag
+            numberOfEnvs={numberOfEnvs}
+            gitSync={gitSync}
+            toggleFeatureFlag={toggleFeatureFlag}
+            governance={governance}
+            cell={cell}
+            refetchFlags={refetchFlags}
+          />
+        )
+      },
+      {
+        Header: getString('details').toUpperCase(),
+        accessor: row => row.kind,
+        width: '22%',
+        Cell: RenderColumnDetails
+      },
+      {
+        Header: getString('status').toUpperCase(),
+        accessor: 'status',
+        width: '21%',
+        Cell: function StatusCell(cell: Cell<Feature>) {
+          const metrics = enableMetricsEndpoint
+            ? featureMetrics?.find(metric => metric.identifier === cell.row.original.identifier)
+            : cell.row.original
+
+          return (
+            <>
+              {metrics ? (
+                <FlagStatus
+                  status={metrics?.status?.status as FeatureFlagStatus}
+                  lastAccess={metrics?.status?.lastAccess as unknown as number}
+                />
+              ) : (
+                <Text icon="spinner" iconProps={{ size: 14 }}>
+                  {getString('cf.loading')}
+                </Text>
+              )}
+            </>
+          )
```

Comment:
Probably just legacy code, but pretty sure we don't need the fragment here
```suggestion
          return metrics ? (
            <FlagStatus
              status={metrics?.status?.status as FeatureFlagStatus}
              lastAccess={metrics?.status?.lastAccess as unknown as number}
            />
          ) : (
            <Text icon="spinner" iconProps={{ size: 14 }}>
              {getString('cf.loading')}
            </Text>
          )
```